lychee-lib/src/filter/mod.rs

```diff
@@ -1,72 +1,140 @@
 mod excludes;
 mod includes;
 
+use std::net::IpAddr;
+
 pub use excludes::Excludes;
 pub use includes::Includes;
 
 use crate::uri::Uri;
 
+/// Pre-defined exclusions for known false-positives
+static FALSE_POSITIVE_PAT: &[&str] = &[r"http://www.w3.org/1999/xhtml"];
```

Comment:
Note: this line is also still in `excludes.rs`.

Reply:
I removed that from `exclude.rs`.